apps/src/netsim/NetSimPacketEditor.js

```diff
@@ -117,7 +117,7 @@ var NetSimPacketEditor = module.exports = function (initialConfig) {
    * @type {Number}
    * @private
    */
-  this.maxPacketSize_ = initialConfig.maxPacketSize || Infinity;
+  this.maxPacketSize_ = initialConfig.maxPacketSize || 8192;
```

Comment:
FYI, this value is measured in bits, not bytes.  8192 therefore represents 1024 characters.  If we reduce this to 1024 bits that's only 128 characters, smaller than a tweet:

> Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer non lectus ut dolor porta congue. Nulla sodales facilisis amet.
> **19 words, 128 bytes, 1024 bits of Lorem Ipsum**

While usable, I think it would feel very restrictive.